index.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Advanced iCal Export
+ * Plugin URI:        https://theeventscalendar.com/extensions/advanced-ical-export/
+ * GitHub Plugin URI: https://github.com/mt-support/extension-template
+ * Description:       The extension gives you advanced export possibilities through the iCal feed
+ * Version:           1.0.0
+ * Extension Class:   Tribe__Extension__Advanced_iCal_Export
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-advanced-ical-export
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( 'Tribe__Extension__Advanced_iCal_Export' )
+) {
```

Comment:
I would recommend formatting this in one single line:
`if ( is_admin() && current_user_can( 'activate_plugins' ) ) {`